tmt/steps/report/reportportal.py

```diff
@@ -54,6 +66,15 @@ def _str_env_to_default(option: str, default: Optional[str]) -> Optional[str]:
     return str(os.getenv(env_var))
 
 
+def _pattern_list_env_to_default(option: str, default: list[Pattern[str]]) -> list[Pattern[str]]:
+    env_var = 'TMT_PLUGIN_REPORT_REPORTPORTAL_' + option.upper()
+    if env_var not in os.environ or os.getenv(env_var) is None:
+        return default
+    return [
+        re.compile(item.strip()) for item in str(os.getenv(env_var)).split(',') if item.strip()
+    ]
```

Comment:
I wonder, if we should not guard this by possible issues compiling the regular expression:

```
      try:
          return [
              re.compile(item.strip())
              for item in str(os.getenv(env_var)).split(',')
              if item.strip()
          ]
      except re.error as e:
          raise tmt.utils.GeneralError(
              f"Invalid regex pattern in environment variable {env_var}: {e}"
          )
```

Reply:
LGTM